Solutions/Recorded Future/Workbooks/Recorded Future - Activelly Communicating C&C IPs to DNS Events - Correlation&Threat Hunting.json

```diff
@@ -0,0 +1,252 @@
+        "size": 0,
+        "exportFieldName": "DestinationIP",
+        "exportParameterName": "IOC",
+        "queryType": 0,
+        "resourceType": "microsoft.operationalinsights/workspaces",
+        "gridSettings": {
+          "formatters": [
+            {
+              "columnMatch": "Risk",
+              "formatter": 18,
+              "formatOptions": {
+                "thresholdsOptions": "colors",
+                "thresholdsGrid": [
+                  {
+                    "operator": ">=",
+                    "thresholdValue": "90",
+                    "representation": "redBright",
+                    "text": "{0}{1}"
+                  },
+                  {
+                    "operator": ">=",
+                    "thresholdValue": "65",
+                    "representation": "orange",
+                    "text": "{0}{1}"
+                  },
+                  {
+                    "operator": ">=",
+                    "thresholdValue": "25",
+                    "representation": "yellow",
+                    "text": "{0}{1}"
+                  },
+                  {
+                    "operator": "Default",
+                    "thresholdValue": null,
+                    "representation": "blue",
+                    "text": "{0}{1}"
+                  }
+                ]
+              }
+            }
+          ]
+        },
+        "sortBy": []
+      },
+      "customWidth": "70",
+      "name": "query - 3"
+    },
+    {
+      "type": 3,
+      "content": {
+        "version": "KqlItem/1.0",
+        "query": "ThreatIntelligenceIndicator\n| where TimeGenerated {Threat_Intelligence_Time_Range:query}\n| where Description contains 'Recorded Future - IP - Activelly Communicating C&C Server'\n| where NetworkIP == \"{IOC}\"\n| where ExpirationDateTime > now()\n| extend Evidence=parse_json(AdditionalInformation)['EvidenceDetails']\n| take 1\n| mv-expand Evidence\n| project Rules = Evidence['Rule'], Evidence_String = Evidence['EvidenceString'], Criticality = Evidence['Criticality']\n| sort by toint(Criticality) desc",
+        "size": 0,
+        "queryType": 0,
+        "resourceType": "microsoft.operationalinsights/workspaces"
+      },
+      "name": "query - 5"
+    }
+  ],
+  "fromTemplateId": "sentinel-UserWorkbook",
```

Comment:
Please update the value of fromTemplateId something other than sentinel-UserWorkbook. Because it is a default value.